java-analyzer-bundle.core/src/main/java/io/konveyor/tackle/core/internal/symbol/CustomASTVisitor.java

```diff
@@ -162,7 +162,7 @@ public boolean visit(ClassInstanceCreation node) {
                 // get fqn of the method being called
                 ITypeBinding declaringClass = binding.getDeclaringClass();
                 if (declaringClass != null) {
-                    String fullyQualifiedName = declaringClass.getQualifiedName() + "." + binding.getName();
+                    String fullyQualifiedName = declaringClass.getQualifiedName();
```

Comment:
During testing for this, I noticed that the Qualified Name from Class was the full name (java.io.File) and adding the binding made it (java.io.File.File) it still mateched the queries but this is more correct